src/robot/libraries/dialogs_ipy.py

```diff
@@ -12,32 +12,183 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import wpf
```

Comment:
Good catch. It's not actually used. It only provides [loading of XAML files] (https://github.com/IronLanguages/main/blob/ipy-2.7-maint/Languages/IronPython/IronPython.Wpf/wpf.cs) and my very first round of code used a XAML file rather than doing the layout programmatically as it is now.

We can drop it.